app/views/static_pages/home/_curriculum.html.erb

```diff
@@ -9,12 +9,14 @@
         <div class="row curriculum-details-grid">
 
           <% available_courses.each do |course| %>
-          <div class="curriculum-details-tile align-items-center">
-            <%= image_tag course[:badge_image_url], alt: course[:badge_alt_text] %>
-            <h4 class="bold text-uppercase">
-              <%= course[:title] %>
-            </h4>
-          </div>
+            <div class="curriculum-details-tile align-items-center">
+              <%= link_to `#{course[:path]}`, `#{course[:path]}` %>
```

Comment:
I think you may have got confused with javascript and ruby string interpolation syntax here. Javascript uses back ticks for interpolation and ruby uses double quotes. In ruby back ticks run system commands, I'm not sure how this works in the view.

Also you're missing a `do` at the end of the line, have you tested this on your local machine.

In any case you don't need interpolation here as `course[:path]` already returns a string. Changing this line to this should be all thats needed: `<%= link_to course[:path] do %>`

Reply:
updated the changes and also tested on local machine :)[test-gif](https://imgur.com/L7dpOpp)